### ISSUE TITLE
Cannot insert Chinese when using Shift + Command + C mode

### ISSUE THREAD

**Issue by verbess** [p0deje/Maccy#689]
### Before Submitting Your Bug Report

- [X] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [X] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

0.29.1

### macOS Version

13.6.1

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":35}";
    "KeyboardShortcuts_popup" = "{\\"carbonModifiers\\":768,\\"carbonKeyCode\\":8}";
    "LaunchAtLogin__hasMigrated" = 1;
    "NSStatusItem Preferred Position Item-0" = 469;
    "NSWindow Frame com.sindresorhus.Preferences.FrameAutosaveName" = "503 152 398 646 0 0 1440 875 ";
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "821 379 513 399 0 0 1440 875 ";
    SUEnableAutomaticChecks = 0;
    SUHasLaunchedBefore = 1;
    SUSendProfileInfo = 0;
    enabledPasteboardTypes =     (
        "public.file-url",
        "public.html",
        "public.png",
        "public.utf8-plain-text",
        "public.tiff",
        "public.rtf"
    );
    ignoredPasteboardTypes =     (
        "net.antelle.keeweb",
        "de.petermaurer.TransientPasteboardType",
        "com.typeit4me.clipping",
        "com.agilebits.onepassword",
        "Pasteboard generator type"
    );
    menuIcon = maccy;
    migrations =     {
        "2020-04-25-allow-custom-ignored-types" = 1;
        "2020-06-19-use-keyboardshortcuts" = 1;
        "2020-09-01-ignore-keeweb" = 1;
        "2021-02-20-allow-to-customize-supported-types" = 1;
        "2021-06-28-add-title-to-history-item" = 1;
        "2021-10-16-remove-dynamic-pasteboard-types" = 1;
        "2022-08-01-rename-suppress-clear-alert" = 1;
        "2022-11-14-add-html-rtf-to-supported-types" = 1;
        "2023-01-22-add-regexp-search-mode" = 1;
    };
    showRecentCopyInMenuBar = 0;
}
```

### Description

Cannot insert Chinese when using Shift + Command + C mode.

- When clicking on the menu bar icon, it works fine.

<img width="789" alt="click_on_menu_bar_icon" src="https://github.com/p0deje/Maccy/assets/28396652/a9f619ef-ea35-4e40-9c36-1c2f7ae336bd">

- However, when using Shift + Command + C mode, the candidate word window does not display.

<img width="812" alt="use_shift_command_c_mode" src="https://github.com/p0deje/Maccy/assets/28396652/9c61a6c2-d5fc-485f-8c86-9be0b7cc1ec2">

### Steps to Reproduce

Here are the Chinese words for your test:
- 你好 (ni hao, means hello).
- 世界 (shi jie, means world).

Steps to Reproduce:
1. use Shift + Command + C.
2. input "nihao".

**Comment by p0deje** [comment]
@verbess Thank you for reporting!

1. Do I need to add some keyboarding input to reproduce the issue?
2. Does it work fine in Maccy 0.28 for you?

**Comment by p0deje** [comment]
Should be fixed in 0.29.2.

**Comment by verbess** [comment]
> @verbess Thank you for reporting!
> 
> 1. Do I need to add some keyboarding input to reproduce the issue?
> 2. Does it work fine in Maccy 0.28 for you?

1. You need to add Simplified Chinese and Simplified Pinyin input methods in the MacOS system.
2. I haven't paid much attention to this feature before, so I'm not sure how it performs in Maccy 0.28.
